### PR TITLE
create e2e tests for logo cloud component

### PR DESCRIPTION
**Pull request by marcell-pal-ds**

### CODE REVIEW

File: tests/content/src/main/content/jcr_root/content/howlite-test/pages/Logo-cloud/.content.xml
Comment (ZofiaZ) [inline tests/content/src/main/content/jcr_root/content/howlite-test/pages/Logo-cloud/.content.xml:101]: let's add empty state as well (to check if placeholder is rendered correctly)

File: tests/end-to-end/tests/logo-cloud.cy.ts
Comment (ZofiaZ) [inline tests/end-to-end/tests/logo-cloud.cy.ts:60]: we don't need to test image component (it should be tested in image.cy instead). I think all we could do here is:
- open logo cloud dialog
- take s percysnapshot
- change grid values to for example: 10 cols width, 1 col offset (let's wait for grid PR to be merged)
- save
- check json file

Comment (marcell-pal-ds) [reply tests/end-to-end/tests/logo-cloud.cy.ts:60]: Alright, I'm waiting for this fix until your grid PR will be merged.

Comment (ZofiaZ) [reply tests/end-to-end/tests/logo-cloud.cy.ts:60]: @marcell-pal-ds I've just merged new grid so we can use offset properties now (smOffset, mdOffset and lgOffset)

File: tests/end-to-end/tests/logo-cloud.cy.ts
Comment (ZofiaZ) [inline tests/end-to-end/tests/logo-cloud.cy.ts:54]: let's add a TODO comment here - for selecting offset and width from dropdown (once WS is ready with test-ids)

File: tests/end-to-end/tests/logo-cloud.cy.ts
Comment (ZofiaZ) [inline tests/end-to-end/tests/logo-cloud.cy.ts:31]: we could use dedicated percySnaphotPreview method now (that excludes header and footer), same for editor and dialog. Please see my merged PR: https://github.com/websight-io/howlite/pull/54

**Review by ZofiaZ — APPROVED** [review]

**Pull request merged**
